app/models/manageiq/providers/ibm_cloud/inventory.rb

```diff
@@ -2,4 +2,13 @@ class ManageIQ::Providers::IbmCloud::Inventory < ManageIQ::Providers::Inventory
   require_nested :Collector
   require_nested :Parser
   require_nested :Persister
+
+  def self.parsed_manager_name(target)
+    case target
+    when InventoryRefresh::TargetCollection
+      'VPC::TargetCollection'
```

Comment:
NOTE: I think we can handle this in core so that we don't have to do this here, but this works as a placeholder (it is going to be an issue for power when they get to targeted refresh)

Reply:
https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/219/files#diff-c05dc858a46b6df73a95914c384303c4d3caf2f692d14814256330bfea18c8b3R10